src-java/network-topology/network-storm-topology/src/test/java/org/openkilda/wfm/topology/network/controller/bfd/AbstractBfdActionTest.java

```diff
@@ -45,8 +45,8 @@
             .logicalPortNumber(alphaLogicalEndpoint.getPortNumber())
             .udpPortNumber(BfdPortFsm.BFD_UDP_PORT)
             .discriminator(1001)
-            .intervalMs(BfdPortFsm.bfdPollInterval)
-            .multiplier(BfdPortFsm.bfdFailCycleLimit)
+            .intervalMs(350)
```

Comment:
Why did you use literals?

Reply:
Because there are no(since this commit) corresponding constants that keep default values for these fields and because this is the test object.